src/v3/src/components/Widget/index.tsx

```diff
@@ -143,8 +146,19 @@ export const Widget: FunctionComponent<WidgetProps> = (widgetProps) => {
         return;
       }
       const transaction = await authClient.idx.start({
-        stateHandle: stateToken,
+        stateHandle,
       });
+      const hasError = !transaction.requestDidSucceed && transaction.messages?.some(
+        (message) => message.class === MessageType.ERROR.toString(),
+      );
```

Comment:
Would it be safer to use: 
```
const hasError = !transaction.requestDidSucceed || transaction.messages?.some(
        (message) => message.class === MessageType.ERROR.toString(),
      );
```

I am not sure that each time a transaction fails here it will accompany a message or vice versa when it fails with a message if the requestDidSucceed flag will be flipped to false. Just to cover all bases I think using the `||` would suffice.